core/sql/SqlCompilerDebugger/mk.sh

```diff
@@ -21,6 +21,9 @@
 export PATH=$QT_TOOLKIT/bin:$PATH
 export LD_LIBRARY_PATH=$QT_TOOLKIT/lib:$LD_LIBRARY_PATH
 
+# Important: QT_TOOLKIT must not be bundled in Trafodion build
+#     QT License is not compatible with ASF license policy
+
 
 qmake -o Makefile SqlCmpDbg.pro
 make
```

Comment:
I guess, if someone wants to use the debugger, then they will have to figure out how to download the toolkit and get it working.  This is probably okay.  If it become important, then someone can create a JIRA and we can work on a change that will make this process easier.